src/opening-period/form/OpeningPeriodForm.test.tsx

```diff
@@ -48,15 +57,14 @@ const testDatePeriodOptions: UiDatePeriodConfig = {
 
 const testResourceId = 1186;
 const testDatePeriodId = 1186;
-const groupIdA = 1;
-const groupIdB = 2;
+const groupIdA = 100;
+const groupIdB = 200;
 const timeSpanGroupA = {
   id: groupIdA,
   period: testDatePeriodId,
   rules: [
     {
       id: 1,
-      group: groupIdA,
```

Comment:
Why are these deleted? Are the groupIds applied somewhere else?

Reply:
I do not remember anymore but I think we should send them back 🤔

Reply:
I fixed it so that the group-id will be passed as form-data. Every new rule or timespan will get current group id if it has one.